checks/evaluation/permissions/GitHubWorkflowPermissionsStepsNoWrite.yml

```diff
@@ -0,0 +1,16 @@
+short: Checks that GitHub workflows do not have steps with dangerous write permissions
+desc: This rule checks that GitHub workflows do not have steps with dangerous write permissions
+motivation: >
+  TODO: attack surface is the Actions called in the steps, and its dependencies.
+
+implementation: >
+  The rule is implemented by checking whether the `permissions` keyword is given non-write permission for the following
+  scopes: X, X, X.
```

Comment:
@joycebrum do you want to take a stab at this one?

Reply:
fyi @olivekl we're revamping the description and remediation

Reply:
Sure, I'll try to suggest something by tomorrow!